.rhcicd/pr_check.sh

```diff
@@ -15,22 +15,26 @@ COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-fronten
 # --------------------------------------------
 # Options that must be configured by app owner
 # --------------------------------------------
-IQE_PLUGINS="hms-provisioning"
-IQE_MARKER_EXPRESSION="smoke"
-IQE_FILTER_EXPRESSION=""
+export IQE_PLUGINS="provisioning"
+export IQE_MARKER_EXPRESSION="ui and smoke"
+export IQE_FILTER_EXPRESSION=""
+export IQE_ENV="ephemeral"
+export IQE_SELENIUM="true"
+export IQE_CJI_TIMEOUT="30m"
+export DEPLOY_TIMEOUT="900"  # 15min
+export DEPLOY_FRONTENDS="true"
 
 set -exv
 # source is preferred to | bash -s in this case to avoid a subshell
 source <(curl -sSL $COMMON_BUILDER/src/frontend-build.sh)
-BUILD_RESULTS=$?
 
-# Stubbed out for now
-mkdir -p $WORKSPACE/artifacts
-cat << EOF > $WORKSPACE/artifacts/junit-dummy.xml
-<testsuite tests="1">
-    <testcase classname="dummy" name="dummytest"/>
-</testsuite>
-EOF
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+# shellcheck source=/dev/null
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
 
-# teardown_docker
-exit $BUILD_RESULTS
+# Run ui tests
+source "${CICD_ROOT}/deploy_ephemeral_env.sh"
+export COMPONENT_NAME="provisioning-backend"
```

Comment:
so we test frontend first and then backend as well? :thinking:

Reply:
So the first `COMPONENT_NAME` is used in order to get the eph env. It's passed as `provisioning-frontend=<sha>` in bonfire deploy

The second `COMPONENT_NAME` is used to run the deploy-iqe-cji, Which is backend. And we pass --selenium to let it know it's a UI.

Reply:
Thanks for the info! :tada: